package.json

```diff
@@ -8,9 +8,9 @@
     "test": "test"
   },
   "scripts": {
-    "coverage": "nyc --reporter=lcov --reporter=text --reporter=json-summary npm test",
-    "lint": "eslint src/ test/",
-    "test": "mocha"
+    "coverage": "./node_modules/.bin/nyc --reporter=lcov --reporter=text --reporter=json-summary npm test",
```

Comment:
This shouldn't be necessary, as npm will look into `node_modules` for the dependencies.

Reply:
Neither with yarn nor with npm can I call the scripts after removing the `./node_modules/.bin/<binary>` part. You may want to verify you do not actually have them installed globally (`%appdata%/npm` if windows) when trying. This is what I get:

```bash
Favna@Stormfortress MINGW64 /e/UserFiles/DevProjects/PubSubJS (master)
$ yarn coverage
yarn run v1.6.0
$ nyc --reporter=lcov --reporter=text --reporter=json-summary npm test
'nyc' is not recognized as an internal or external command,
operable program or batch file.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

Favna@Stormfortress MINGW64 /e/UserFiles/DevProjects/PubSubJS (master)
$ npm coverage

Usage: npm <command>

where <command> is one of:
    access, adduser, audit, bin, bugs, c, cache, ci, cit,
    completion, config, create, ddp, dedupe, deprecate,
    dist-tag, docs, doctor, edit, explore, get, help,
    help-search, hook, i, init, install, install-test, it, link,
    list, ln, login, logout, ls, outdated, owner, pack, ping,
    prefix, profile, prune, publish, rb, rebuild, repo, restart,
    root, run, run-script, s, se, search, set, shrinkwrap, star,
    stars, start, stop, t, team, test, token, tst, un,
    uninstall, unpublish, unstar, up, update, v, version, view,
    whoami

npm <command> -h     quick help on <command>
npm -l           display full usage info
npm help <term>  search for help on <term>
npm help npm     involved overview

Specify configs in the ini-formatted file:
    C:\Users\Favna\.npmrc
or on the command line via: npm <command> --key value
Config info can be viewed via: npm help config

npm@6.0.0 C:\Users\Favna\AppData\Roaming\npm\node_modules\npm

Did you mean this?
    create
```

Reply:
I don't have `nyc` installed as a global and I have no issues running any of the scripts on macOS.

This is the first I've heard of people having this particular issue with npm scripts (on any platform), so this might be specific to your setup. Would you mind investigating further, or provide links to documentation?